framework/platform/bpl/uci/cfg/bpl_cfg.cpp

```diff
@@ -459,5 +459,19 @@ bool cfg_get_unfriendly_device_max_timelife_delay_days(
     return true;
 }
 
+bool bpl_cfg_get_wpa_supplicant_ctrl_path(const std::string &iface, std::string &wpa_ctrl_path)
+{
+    std::string path = "/var/run/wpa_supplicant/";
+    wpa_ctrl_path    = path + iface;
+    return true;
+}
+
+bool bpl_cfg_get_hostapd_ctrl_path(const std::string &iface, std::string &hostapd_ctrl_path)
+{
+    std::string path  = "/var/run/hostapd/";
+    hostapd_ctrl_path = path + iface;
+    return true;
+}
```

Comment:
Does i correctly understand that for case of using UCI instead of `prplmesh_platform_db` wpa/hostapd ctrl path will be still hard-coded?
Maybe add separate task for real get wpa/hostapd ctrl path from UCI variables which are added by prplmesh init script or OpenWrt package?

Reply:
As I understood, this is a common path for OpenWRT and we can keep it as is.